include/librealsense2/dds/topics/device-info/device-info-msg.h

```diff
@@ -12,11 +12,7 @@ class device_info
 {
 public:
     using type = raw::device_infoPubSubType;
-
-    std::string name;
-    std::string serial;
-    std::string product_line;
-    bool locked;
+    static constexpr char* const DEVICE_INFO_TOPIC_NAME = "realsense/device-info";
```

Comment:
If it's in here, then why is the name so cumbersome? Why not simply `device_info::TOPIC_NAME`?

Reply:
Done